paper/paper.md

```diff
@@ -250,7 +250,7 @@ Currently, plugins for CI services like TravisCI, CircleCI, Jenkins, and Gitlab-
 In this section, we present a case study demonstrating how the Popper workflow engine allows reproducing and scaling workflows in different computing environments.
 This study aims to emphasize on how Popper can help in mitigating the reproducibility issues and make life easier for researchers and developers.
 
-The high-level steps that are required for building reproducible workflows with Popper usually consists of the following steps:
+The general paradigm for building reproducible workflows with Popper usually consists of the following steps:
```

Comment:
we use "paradigm" multiple times to refer to the container-native paradigm. it would be preferable to use something else. "high-level procedure", "general outline" or something like that